spec.html

```diff
@@ -29931,11 +29926,8 @@ <h1>
             <dd>_constructor_ is the constructor function that is performing this action. _newTarget_ is the constructor that `new` was initially applied to. _args_ is the argument values that were passed to _constructor_.</dd>
           </dl>
           <emu-alg>
-            1. Assert: The execution context stack has at least two elements.
-            1. Let _callerContext_ be the second to top element of the execution context stack.
-            1. Let _callerRealm_ be _callerContext_'s Realm.
             1. Let _calleeRealm_ be the current Realm Record.
```

Comment:
The _calleeRealm_ might be clearer as _currentRealm_ at this point, without a _callerRealm_ to contrast to.